src/ZString/FormatHelper.cs

```diff
@@ -48,17 +48,17 @@ private static void FormatToRightJustify<TBufferWriter, T>(ref TBufferWriter sb,
             if (typeof(T) == typeof(string))
             {
                 var s = Unsafe.As<string>(arg);
-                int padding = width - s.Length;
+                int padding = width - s!.Length;
```

Comment:
Redundant when the initialization of `s` is adjusted - `var s = Unsafe.As<string>(arg)!;`